app/controllers/boards_controller.rb

```diff
@@ -171,6 +173,17 @@ def require_permission
     end
   end
 
+  def boards_from_relation(relation)
+    sql = <<~SQL.squish
+      boards.*,
+      (SELECT tagged_at FROM posts WHERE posts.board_id = boards.id ORDER BY posts.tagged_at DESC LIMIT 1) AS tagged_at
```

Comment:
I think this is just
```suggestion
      (SELECT MAX(tagged_at) FROM posts WHERE posts.board_id = boards.id) AS tagged_at
```

Reply:
Hm! possibly, yeah